src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Recipes/AzureADSettingsStep.cs

```diff
@@ -27,14 +30,13 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
             }
 
             var model = context.Step.ToObject<AzureADSettingsStepModel>();
-            var settings = await _azureADService.LoadSettingsAsync();
 
-            settings.AppId = model.AppId;
-            settings.TenantId = model.TenantId;
-            settings.DisplayName = model.DisplayName;
-            settings.CallbackPath = model.CallbackPath;
+            _azureADSettings.AppId = model.AppId;
+            _azureADSettings.TenantId = model.TenantId;
+            _azureADSettings.DisplayName = model.DisplayName;
+            _azureADSettings.CallbackPath = model.CallbackPath;
```

Comment:
I don't think we should override the options here on import. The service will update the settings, so after a tenant restart, they'll be applied, if necessary. Same for `MicrosoftAccountSettingsStep`, `TwitterSettingsStep`

Reply:
Again to avoid the confusion we should be clear what should be exported and what shouldn't. May be we need to document this or show an alter to the user on admin settings

Reply:
I don't see an issue, really. With this code, the import override both the currently applied config from `IOptions` as well as the site settings. After a tenant restart, the site settings might not actually take effect though. So, import shouldn't circumvent the normal loading of settings.